### PR TITLE
clean code

### PR DESCRIPTION
**Pull request by varyuan**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: core/src/main/java/com/alibaba/nacos/core/control/TpsControlConfig.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public class TpsControlConfig {
      * tps control is enabled.
      * @return true/false.
      */
-    public static final boolean isTpsControlEnabled() {
+    public static boolean isTpsControlEnabled() {
```
Comment (KomachiSion) [inline core/src/main/java/com/alibaba/nacos/core/control/TpsControlConfig.java:33]: Why remove final

File: plugin/control/src/main/java/com/alibaba/nacos/plugin/control/ruleactivator/RuleStorageProxy.java [hunk 2]
```diff
@@ -75,7 +75,7 @@ public RuleStorage getExternalStorage() {
         return externalRuleStorage;
     }
     
-    public static final RuleStorageProxy getInstance() {
+    public static RuleStorageProxy getInstance() {
```
Comment (KomachiSion) [inline plugin/control/src/main/java/com/alibaba/nacos/plugin/control/ruleactivator/RuleStorageProxy.java:78]: Why remove final

File: plugin/control/src/main/java/com/alibaba/nacos/plugin/control/tps/nacos/LocalSimpleCountBarrierCreator.java [hunk 3]
```diff
@@ -33,7 +33,7 @@ public class LocalSimpleCountBarrierCreator implements RuleBarrierCreator {
     public LocalSimpleCountBarrierCreator() {
     }
     
-    public static final LocalSimpleCountBarrierCreator getInstance() {
+    public static LocalSimpleCountBarrierCreator getInstance() {
```
Comment (KomachiSion) [inline plugin/control/src/main/java/com/alibaba/nacos/plugin/control/tps/nacos/LocalSimpleCountBarrierCreator.java:36]: Why remove final

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: plugin/control/src/main/java/com/alibaba/nacos/plugin/control/tps/nacos/LocalSimpleCountBarrierCreator.java (hunk 3 above)
Comment (varyuan) [reply plugin/control/src/main/java/com/alibaba/nacos/plugin/control/tps/nacos/LocalSimpleCountBarrierCreator.java]: final防止子类重写方法, 静态方法不能被@Override, 加上也没错,但是应该也不会写出子类声明相同的方法签名的静态方法吧, 改了

File: plugin/control/src/main/java/com/alibaba/nacos/plugin/control/ruleactivator/RuleStorageProxy.java [hunk 4]
```diff
@@ -74,7 +74,7 @@ public RuleStorage getLocalDiskStorage() {
     public RuleStorage getExternalStorage() {
         return externalRuleStorage;
     }
-    
+
```
Comment (KomachiSion) [inline plugin/control/src/main/java/com/alibaba/nacos/plugin/control/ruleactivator/RuleStorageProxy.java:77]: Why remove indent?

Please use nacos code style to reformat.

File: plugin/control/src/main/java/com/alibaba/nacos/plugin/control/tps/nacos/LocalSimpleCountBarrierCreator.java [hunk 5]
```diff
@@ -32,7 +32,7 @@ public class LocalSimpleCountBarrierCreator implements RuleBarrierCreator {
     
     public LocalSimpleCountBarrierCreator() {
     }
-    
+
```
Comment (KomachiSion) [inline plugin/control/src/main/java/com/alibaba/nacos/plugin/control/tps/nacos/LocalSimpleCountBarrierCreator.java:35]: Why remove indent?

**Review by KomachiSion — CHANGES_REQUESTED** [review]

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**